extensions/tmtv/src/Panels/PanelPetSUV.tsx

```diff
@@ -171,7 +195,7 @@ export default function PanelPetSUV({ servicesManager, commandsManager }) {
               }}
             />
             <Input
-              label={t('Half Life')}
+              label={t('Half Life (Seconds)')}
```

Comment:
If you are using the SI units elsewhere, then (s) would be the right value here.